src/org/opendatakit/briefcase/export/PullBeforeOverrideOption.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2018 Nafundi
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.opendatakit.briefcase.export;
+
+import java.util.Optional;
+import java.util.stream.Stream;
+
+public enum PullBeforeOverrideOption {
+  INHERIT("Use defaults", null), PULL("Pull before export", true), DONT_PULL("Do not pull before export", false);
```

Comment:
“Use defaults” (plural) suggests there are more than one. I suggest “Use default” or “Use Export Page setting”. Or maybe it should be called the Export Pane or Panel or Screen or Tab? (I don’t know if we’ve described the `JPanel`s in documentation.)